python/veles/objects/base.py

```diff
@@ -100,3 +104,18 @@ def __repr__(self):
         return 'id: {} name: {}'.format(self.id, self.name)
 
     __str__ = __repr__
+
+    def create_chunk(self, name, start=None, end=None, size=None,
+                     comment='', chunk_type='', update_cursor=True):
+        if start is None:
+            start = self._cursor
+
+        if end is None and size is None:
+            raise ValueError(
+                'You need to specify either size or end of chunk')
+        if end is None:
+            end = start + size
+        if update_cursor:
+            self._cursor = end
```

Comment:
Hmm now that I think that it is better if we update cursor after we create chunk - that way if something fails and we want to retry from IPython we still have the old cursor value.